department-of-veterans-affairs/vets-website
# Update single facility messaging in new appt flow

## Description
This PR
- Updates the single valid facility scenario for the flat facility page to match new design
- Fixes some flaky tests I encountered

How to test:
- Open http://localhost:3001/health-care/schedule-view-va-appointments/appointments/new-appointment
- Start flow, choose mental health
- See single facility available design

## Testing done
Local and unit testing

## Screenshots
![Screen Shot 2021-02-22 at 3 33 00 PM](https://user-images.githubusercontent.com/634932/108766529-44973f00-7523-11eb-8527-0b52c4a49866.png)

## Acceptance criteria
- [ ] New design for single facility scenario is correct

## Definition of done
